fuzzer/cascade/fuzzerstate.py

```diff
@@ -97,6 +97,28 @@ def reset(self):
         # Coordinates of the FPU enable/disable instructions. Only used in program reduction.
         self.fpuendis_coords = []
 
+    def add_instruction(self, new_instrobjs):
```

Comment:
Please add some comments for the new functions to tell quickly their effect, and in which contexts they should or should not be used.